src/main/java/vexatos/tgregworks/item/ItemTGregPart.java

```diff
@@ -45,18 +46,16 @@ public String getItemStackDisplayName(ItemStack stack) {
 		NBTTagCompound data = TGregUtils.getTagCompound(stack);
 		String matName;
 		if(!data.hasKey("material") || Materials.get(data.getString("material")) == Materials._NULL) {
-			matName = "Unknown";
+			matName = StatCollector.translateToLocal("tgregworks.materials.unknown");
 		} else {
-			matName = Materials.get(data.getString("material")).mDefaultLocalName;
+			matName = Materials.get(data.getString("material")).mLocalizedName;
 		}
 
-		matName = matName + " " + type.getPartName();
-		//String material = StatCollector.translateToLocal("tgregworks.parttype." + matName);
-		//String name = StatCollector.translateToLocal("tgregworks.toolpart." + PartTypes.getFromID(stack.getItemDamage()) + "." + matName);
-		//name = name.replaceAll("%%material", material);
+		String name = StatCollector.translateToLocal("tgregworks.toolpart." + type.getID());
```

Comment:
Instead of the ID, one could use something like `type.name()`. Would probably be more sane in the lang file.

Reply:
Ok, I will fix this in next commit.